test/compiler/parsing/fixtures/watf/wast-identifier-to-index/actual.wast

```diff
@@ -0,0 +1,9 @@
+(func (param $c f64) (param $a i32) (param $b i64)
+  (get_local $a)
+  (get_local $b)
+  (get_local $c)
+  (get_local $a)
+
+  (set_local $a (i32.const 1))
+  (tee_local $a (i32.const 1))
+)
```

Comment:
Sorry for unrelated tests modification, this is the interesting one.

Basically replacing `(get_local $c)` with its offset in the locals: `(get_local 0)`.